S11-repository/curli-install.t

```diff
@@ -16,16 +18,18 @@ $path.IO.mkdir;
     is $*REPO.short-id, 'inst', 'short-id exposes type of repository';
     is $*REPO.loaded.elems, 0, 'no compilation units were loaded so far';
 }
-my $dist              = Distribution.new(:name<Foo>, :api<1>, :ver(v1.2.3));
-my $non-matching-dist = Distribution.new(:name<Foo>, :api<2>, :ver(v2.3.4));
-my $foo-path = $path.IO.parent.child('Foo.pm').relative;
 
-$*REPO.install($dist,              { Foo => $foo-path });
-$*REPO.install($non-matching-dist, { Foo => $foo-path });
+my $prefix   = $repo-path.IO.parent;
+my %provides = Foo => $prefix.child('Foo.pm').relative($prefix);
```

Comment:
Isn't this just my %provides = Foo => 'Foo.pm';?

Reply:
It could be -- I assumed this way was used originally because it was more explicit about the relation between the path/prefix and the provided file's path.